src/Relation/Binary/Construct/Closure/Reflexive/Properties.agda

```diff
@@ -0,0 +1,59 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties of reflexive closures
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K #-}
+
+module Relation.Binary.Construct.Closure.Reflexive.Properties where
+
+open import Data.Product as Prod
+open import Function
+open import Relation.Binary
+open import Relation.Binary.Construct.Closure.Reflexive
+import Relation.Binary.PropositionalEquality as PropEq
+open PropEq using (_≡_; refl)
+open import Relation.Nullary
+
+module _ {a b ℓ} {A : Set a} {B : Set b} {_~_ : Rel A ℓ} where
```

Comment:
Everything should be indented using only two spaces. You could also consider using the new `variable` keyword for `a,b,l,A,B`.

Reply:
> Everything should be indented using only two spaces.

Done

> You could also consider using the new `variable` keyword for `a,b,l,A,B`.

I couldn't figure out how to do so — if i merely use the obvious `variable` declaration, type-checking fails. Unless you feel strongly i'll just leave it as is.

Reply:
No worries, that's fine.